### ISSUE TITLE
Undergraduate summer project at Sydney University

### ISSUE THREAD

**Issue by JoshMaxwell** [OpenSourceMalaria/OSM_To_Do_List#554]
Hello everybody,

My name is Josh Maxwell and I am an undergraduate student at the University of Sydney. Over the summer I am lucky enough to be working on Open Source Malaria for six weeks, having just completed my second week and returning in January for the last four. 

Over the last two weeks, I have been working on the synthesis of analogues for the series four compounds. The recent compound arising from a biosynthesis (#513) shows very promising potency with the para-phenol, the chemical synthesis (#525) aims to confirm this, we are awaiting potency results (OpenSourceMalaria/Series4#10).

However, phenolic compounds are associated with high levels of phase II metabolism and the potential for the production of toxic metabolites (see [Meanwell, 2011, 54(8), 2529-2591](http://pubs.acs.org/doi/pdf/10.1021/jm1013693)). Hence, to avoid these negative implications but hopefully retain the potency of the phenolic compound, we need to make bio-isosteric replacements at the hydroxyl position in these potent compounds (OpenSourceMalaria/Series4#3)
I have begun the process of making the first of these bio-isosteres, specifically a 3-bromo-pyridine and an aniline. The below reactions were completed successfully for both of the compounds. (ELN link going live soon - edit by Mat: ELN is now available: [https://tinyurl.com/OSM-JoshMaxwell](https://au-mynotebook.labarchives.com/share/Joshua%2520Maxwell%2520Denison/MC4wfDM0NTQ3LzAvVHJlZU5vZGUvMjU4NzEyMTgzfDAuMA==).

![jm1-1](https://user-images.githubusercontent.com/34295412/33699513-b1a7eaf8-db67-11e7-988d-fdcfd022f7a0.jpg)
![jm2-1](https://user-images.githubusercontent.com/34295412/33699514-b1d70bd0-db67-11e7-889e-bca06171e6b1.jpg)

I am currently completing the reduction of both of the esters to make the corresponding diols and am in the process of confirming that the reactions were successful. I look forward to continuing all of this in January!

**Comment by david1597** [comment]
Great start by Josh in the first two weeks. We need to give him more work though! Discussion, and poll, about which compounds we should target next in this series is over at #555.

**Comment by mcoster** [comment]
Great stuff, Josh! Can't wait to see how your project goes :)

**Comment by mattodd** [comment]
ELN now available - link added to Josh's original post.

**Comment by mattodd** [comment]
Relevant literature @MFernflower ? The Br needn't be there in the final compound - it can be used for other things as per @maratsydney comment in OpenSourceMalaria/OSM_To_Do_List#555

**Comment by MFernflower** [comment]
@mattodd I do not mean to be a nitpick but there are mistakes in the ELN synthesis schemes: 
![jm6-1](https://user-images.githubusercontent.com/3164942/33920732-7fe403d8-df8d-11e7-8cdc-f68a7eb991c3.jpg)

**Comment by david1597** [comment]
@MFernflower  Well spotted! That mistake came from copying over from Ed's protocol for that reaction, where the cyclopropylethanol was the alcohol used in the substitution. I think we changed it in the HIRAC (our hazard and risk assessment) but missed it here. We didn't quite get around to starting the reaction...it will be done - and that error fixed - when Josh returns in January.

**Comment by david1597** [comment]
Closing issue.

Josh's ELN remains [here](https://au-mynotebook.labarchives.com/share/Joshua%2520Maxwell%2520Denison/MC4wfDM0NTQ3LzAvVHJlZU5vZGUvMjU4NzEyMTgzfDAuMA==).

The compounds which he worked on ultimately fed into those discussed [here ](https://github.com/OpenSourceMalaria/Series4/issues/3)which were analogues of the[ Pfizer phenol](https://github.com/OpenSourceMalaria/Series4/issues/33) containing the 2-phenylpropan-1-ol fragment. However, work on this series largely halted shortly after Josh left after we realised the compound wasn't the one that had given the potent hit. A few compounds have still trickled through; they can be found through the links above or searching using the 2-phenylpropan-1-ol in DataWarrior.
